ddp-workspace/projects/ddp-singular/src/app/components/pages/participant-list/participant-list.component.ts

```diff
@@ -413,13 +420,23 @@ export class ParticipantsListComponent implements OnInit {
   }
 
   private filterOutAddParticipant(activities: ActivityInstance[]): ActivityInstance[] {
-    return activities.filter(
+    return !!activities ? activities.filter(
```

Comment:
**#minor #optional**

`return !!activities ? activities.filter(...) : [];`
can be shortened like
`return (activities || []).filter(...)`

or just set a default value for the method argument, like

```
private filterOutAddParticipant(activities: ActivityInstance[] = []): ActivityInstance[] {
   return activities.filter(...);
}
```

Reply:
Done